packages/api/tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
@gobengo could we add in here the types for the actual CF workers? I think that brings a ton of benefits. Example:

 https://github.com/nftstorage/nftstorage.link/blob/main/packages/edge-gateway/tsconfig.json#L7

Reply:
We currently have all types for bindings specified in `env.js`, which is not cool. Adding CF Worker types and creating a proper ts file for the bindings injected makes things so much cleaner. You can also see https://github.com/nftstorage/nftstorage.link/blob/main/packages/edge-gateway/src/bindings.d.ts

Reply:
https://github.com/web3-storage/web3.storage/issues/1832

Thanks for the good suggestion. I'll do it when I have a sec